### PR TITLE
MAP-18 - UI for managing mappings

### PR DESCRIPTION
**Pull request by tmarzeion**
DONE

### CODE REVIEW

**Comment by dkayiwa** [comment]
@tmarzeion did you notice the merge conflicts on this?

**Comment by dkayiwa** [comment]
@tmarzeion i also see conflicts in this. :)

**Comment by tmarzeion** [comment]
@rkorytkowski OWA is ready, You can check it out if You want :)
Ill create tests for it soon.
PS. There is no message specified for alert-dialog when setting up mapping, because I didn't know how it should look like, any suggestions?

**Comment by tmarzeion** [comment]
@rkorytkowski Finished

File: omod/src/main/java/org/openmrs/module/metadatamapping/web/rest/MetadataTermMappingResource.java [hunk 1]
```diff
@@ -38,6 +39,8 @@
 	
 	public static final String PARAM_METADATA_UUID = "metadataUuid";
 	
+	public static final String PARAM_DEFINED = "def";
```
Comment (rkorytkowski) [inline omod/src/main/java/org/openmrs/module/metadatamapping/web/rest/MetadataTermMappingResource.java]: Rename to mapped

**Pull request merged**